src/main/kotlin/com/linchpino/core/controller/InterviewController.kt

```diff
@@ -31,4 +36,18 @@ class InterviewController(private val service: InterviewService) {
         val result = service.createInterview(request)
         return ResponseEntity.status(HttpStatus.CREATED).body(result)
     }
+
+    @Operation(summary = "Return page of upcoming interviews for authenticated mentor")
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Upcoming interviews fetched successfully"),
+            ApiResponse(responseCode = "401", description = "User is not authenticated"),
+            ApiResponse(responseCode = "403", description = "Authenticated user is not MENTOR")
+        ]
+    )
+    @GetMapping("/upcoming")
```

Comment:
Do you have any idea for the url mapping when a job seeker wants to get list of the upcoming interviews ?

Reply:
Should we add mentors before upcoming? /mentors/upcoming?
I am open to any suggestions.

Reply:
I think it a good Idea same as this: api/interviews/mentors/upcoming